jbpm-case-mgmt/jbpm-case-mgmt-impl/src/test/resources/org/jbpm/casemgmt/demo/insurance/insurance-rules.drl

```diff
@@ -6,7 +6,7 @@ import org.jbpm.casemgmt.demo.insurance.ClaimReport;
 rule "classify-claim" ruleflow-group "claims"
 
 when 
-    $caseData : CaseData()
+    $caseData : CaseData() from entry-point "insurance-claims.CarInsuranceClaimCase" 
```

Comment:
Doesn't this introduce backwards compatibility issues, and pushes the burden partially to the user as well (just in a different location, now the rules are specifying the entrypoint rather than the rules having to make sure they are independent from other cases if necessary.